src/index.ts

```diff
@@ -29,6 +29,12 @@ const convertStateToEnv = (state: MediaState): Parameters<typeof matches>[1] =>
     return env;
 };
 
+function toCamelCase(str: string) {
+    return str.replace(/^-*/, "").replace(/-([a-z])/g, (match: string) => {
+        return match.charAt(1).toUpperCase() ?? "";
```

Comment:
I was checking `media-query-fns` source code, and I think some aren't just camel cases of the input. Like `monochrome` -> `monochromeBits` ([see here](https://github.com/tbjgolden/media-query-fns/blob/7dae2618b9321f503cbd0a44a202a9190665e80e/lib/matches.ts#L459-L466)), or `resolution` -> `dppx` ([see here](https://github.com/tbjgolden/media-query-fns/blob/7dae2618b9321f503cbd0a44a202a9190665e80e/lib/matches.ts#L474-L479))